timur/lib/client/jsx/components/query/query_table.tsx

```diff
@@ -14,6 +16,10 @@ import {QueryTableColumn} from '../../contexts/query/query_types';
 const useStyles = makeStyles({
   table: {
     minWidth: 650
+  },
+  columnWarning: {
+    color: 'darkred',
```

Comment:
a lighter shade of red would stand out more to me.  I understand that red is super bright to color-typical individuals, but dark red can blend in with black for me!